api/src/utils/Logger.ts

```diff
@@ -0,0 +1,19 @@
+import winston from "winston";
+import { environment } from "..";
+
+const Logger = winston.createLogger({
+  level: environment === "development" ? "debug" : "warn",
+  format: winston.format.combine(
+    winston.format.timestamp({ format: "YYYY-MM-DD HH:mm:ss:ms" }),
+    winston.format.json(),
+    winston.format.colorize({ all: true })
```

Comment:
Q: Will this only apply to `console`, or will it fill log files with ANSI colour escape sequences?